critiquebrainz/frontend/templates/recording/entity.html

```diff
@@ -0,0 +1,140 @@
+{% extends 'base.html' %}
+{% from 'macros.html' import show_avg_rating, entity_rate_form, show_external_reviews with context %}
+
+{% block title %}
+  {{ _('Recording "%(name)s" by %(artist)s', name=recording.name, artist=recording['artist-credit-phrase']) }}
+  - CritiqueBrainz
+{% endblock %}
+
+{% block content %}
+<div class="clearfix">
+  <h2 class="pull-left">
+    {% set artist = [] %}
```

Comment:
I think we can do this a bit better, can we do something like 

```
{{recording.name}} _(by) 
{% for credit in recording['artists'] %}
{% if credit.name %}
    <a href="{{url_for('artist.entity', mbid=credit.id)}}">{{credit.name}}</a>
    {% if credit.join_phrase %}{{credit.join_phrase}}{% endif %}
{% endif %}
```

What does it mean when a credit doesn't have a name and we just append it? (line 20)

Reply:
Done.